builddef-x64.nsi

```diff
@@ -1,7 +1,7 @@
 ; builddef-x64.nsi
 
 !define PRODUCT_VERSION "0.5.2.41"
-!define PACKAGE_VERSION "20000101"
+!define PACKAGE_VERSION "20211102"

```

Comment:
Don't change `PACKAGE_VERSION` it is automatically updated with proper version at build time.